### PR TITLE
checks for cloud license

### PR DESCRIPTION
**Pull request by AjeyHiremath**

### CODE REVIEW

File: ea-defs/config.go [hunk 1]
```diff
@@ -157,3 +158,32 @@ func RequiredEADefsFor(cloud_type string) (res []ibclient.EADefinition) {
...
+	flag, err := CheckLicense(objMgr, "cloud")
+	if err != nil {
+		fmt.Println("error", err)
+		os.Exit(4)
+	}
+	if !flag {
+		fmt.Println("Cloud License does not exist")
+		os.Exit(4)
+	}
+
```
Comment (jkraj) [inline ea-defs/config.go]: nit: remove this line

File: ea-defs/config.go [hunk 2]
```diff
@@ -157,3 +158,32 @@ func RequiredEADefsFor(cloud_type string) (res []ibclient.EADefinition) {
 	}
 	return
 }
+
+//Checks for cloud license in nios
+func CheckForCloudLicense(objMgr *ibclient.ObjectManager) {
+	flag, err := CheckLicense(objMgr, "cloud")
+	if err != nil {
+		fmt.Println("error", err)
```
Comment (jkraj) [inline ea-defs/config.go]: In other place of this file uses 'logrus' use that

File: ea-defs/config.go [hunk 3]
```diff
@@ -157,3 +158,32 @@ func RequiredEADefsFor(cloud_type string) (res []ibclient.EADefinition) {
...
+		os.Exit(4)
+	}
+	if !flag {
+		fmt.Println("Cloud License does not exist")
```
Comment (jkraj) [inline ea-defs/config.go]: Cloud License not available in Infoblox Appliance. Update and try again..

File: ea-defs/config.go (hunk 2 above)
Comment (AjeyHiremath) [reply ea-defs/config.go]: done

File: ea-defs/config.go (hunk 3 above)
Comment (AjeyHiremath) [reply ea-defs/config.go]: done

File: ea-defs/config.go (hunk 1 above)
Comment (AjeyHiremath) [reply ea-defs/config.go]: done

**Review by jkraj — APPROVED** [review]

**Pull request merged**